docs/experimental/extendable-param-files.md

```diff
@@ -0,0 +1,90 @@
+# Using the Extendable Bicep Parameters Feature (Experimental!)
```

Comment:
I don't think you need "(Experimental!)" here since all of these docs are for experimental features